sdk/monitor/Azure.Monitory.Query/src/Models/LogsQueryResultRow.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+
+namespace Azure.Monitory.Query.Models
+{
+    public class LogsQueryResultRow
+    {
+        private readonly Dictionary<string, int> _columns;
+        private readonly JsonElement _row;
+
+        internal LogsQueryResultRow(Dictionary<string, int> columns, JsonElement row)
+        {
+            _columns = columns;
+            _row = row;
+        }
+
+        public int Count => _row.GetArrayLength();
+
+        public int GetInt32(int index) => _row[index].GetInt32();
```

Comment:
It's unclear what these methods do based on their names. Maybe something like `GetInt32ByIndex(int columnIndex)` and `GetInt32ByName(string columnName)`? 

Also, could these be generic? (`GetByIndex<T>(int columnIndex)` and `GetByName<T>(string columnName)`)

Reply:
I was mimicking `DataReader` and `JsonElement` method here:
 https://docs.microsoft.com/en-us/dotnet/api/system.data.datatablereader.getint64?view=net-5.0
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonelement.getint32?view=net-5.0